driver/pace/driver/driver.py

```diff
@@ -13,8 +13,6 @@
 import pace.dsl
 import pace.physics
 import pace.stencils
-import pace.util
-import pace.util.grid
```

Comment:
Same issue here as in diagnostics: we still depend on imports of pace.util and pace.util.grid